src/jira.ts

```diff
@@ -0,0 +1,77 @@
+import * as core from '@actions/core'
+import JiraApi from 'jira-client'
+
+export class Jira extends JiraApi {
+  private jiraIdRegex = /((?<!([A-Za-z]{1,10})-?)[A-Z]+-\d+)/g
```

Comment:
What's with the negative lookbehind?

Reply:
It's taken from https://stackoverflow.com/a/30518972
It prevents something like XX-JIRA-123 and abcJIRA-123 from matching.